static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -69,18 +71,20 @@ class SpanTree extends Component<PropType> {
   }
 
   generateInfoMessage(input: {
+    filteredSpansAbove: EnhancedProcessedSpanType[];
     isCurrentSpanFilteredOut: boolean;
     isCurrentSpanHidden: boolean;
-    numOfFilteredSpansAbove: number;
```

Comment:
I'm now storing the actual span objects array instead of just a count, so we no longer need `numOfFilteredSpansAbove`